src/tools/remote-test-server/src/main.rs

```diff
@@ -72,21 +76,23 @@ fn main() {
 
     let config = Config::parse_args();
 
-    let bind_addr = if cfg!(target_os = "android") || config.remote {
+    let bind_addr = if cfg!(target_os = "android") || cfg!(windows) || config.remote {
         "0.0.0.0:12345"
     } else {
         "10.0.2.15:12345"
     };
 
-    let (listener, work) = if cfg!(target_os = "android") {
-        (t!(TcpListener::bind(bind_addr)), "/data/tmp/work")
+    let listener = t!(TcpListener::bind(bind_addr));
+    let work: PathBuf = if cfg!(windows) {
+        env::var_os("RUST_TEMP").expect("Set RUST_TEMP to your preferred temp folder").into()
+    } else if cfg!(target_os = "android") {
+        "/data/tmp/work".into()
     } else {
-        (t!(TcpListener::bind(bind_addr)), "/tmp/work")
+        "/tmp/work".into()
```

Comment:
Does https://doc.rust-lang.org/nightly/std/env/fn.temp_dir.html do the wrong thing here?

Reply:
Looks like the perfect way, it also unifies windows and non-android-unix :)

The path specified for android seems to differ from what `temp_dir()` returns according to the docs, so I've left that one as-is.